src/PrestaShopBundle/Controller/Admin/Improve/Design/PositionsController.php

```diff
@@ -118,23 +121,24 @@ public function indexAction(Request $request)
                 ]
             ],
             'showModules' => $this->showModules,
+            'selectedModule' => $this->selectedModule,
             'layoutTitle' => $this->trans('Positions', 'Admin.Navigation.Menu'),
             'requireAddonsSearch' => false,
             'requireBulkActions' => false,
             'requireFilterStatus' => false,
             'showContentHeader' => true,
             'enableSidebar' => true,
             'help_link' => $this->generateSidebarLink('AdminModulesPositions'),
-            'selectedModule' => $this->selectedModule,
             'hooks' => $hooks,
             'modules' => $modules,
-            'canMove' => !(Shop::isFeatureActive() && Shop::getContext() != Shop::CONTEXT_SHOP)
+            'canMove' => $this->get('prestashop.adapter.shop.context')->isSingleShopContext()
         ];
     }
 
     /**
      * Transplant a module
-     * @TODO this part isn't finished yet waiting for CQRS
+     *
+     * @TODO this part isn't finished yet, waiting for CQRS!
```

Comment:
it's matter of taste, but imo we should never wait for undefined abstractions to be able to finish concrete implementations ;)